packages/nimble-components/src/select/specs/filerable-select-hld.md

```diff
@@ -0,0 +1,78 @@
+# Filterable Select
+
+## Problem Statement
+
+Our clients have a need of a filterable dropdown component that does not require arbitrary text as a value, but only values available in the dropdown. The filtering will have the following behaviors:
+
+-   The filter text will match _any_ text within the `textContent` of each `ListOption`
```

Comment:
Could you mention here whether or not it will be case sensitive or case insensitive? Are there any localization considerations we need to consider?

Reply:
I've mentioned that it will be case insensitive. I don't think there are any localization considerations beyond what is called out in the `LabelProviderCore` section, as Nimble won't be responsible for any localization needed for text within each option, and ultimately the filtering will work against whatever the rendered text content is (which I believe would be the expected behavior).